tiatoolbox/utils/exceptions.py

```diff
@@ -0,0 +1,8 @@
+"""Custom Errors and Exceptions for TIAToolbox"""
+
+
+class FileNotSupported(Exception):
```

Comment:
instead - word as:  'File format is not supported'

Reply:
Updated in this commit https://github.com/TIA-Lab/tiatoolbox/commit/dd194c4ee46e2bc4f36c73138f8f03515037b1fd